SlowCheetah.JDT/Processors/Attributes/JdtAttributeValidator.cs

```diff
@@ -0,0 +1,85 @@
+﻿namespace SlowCheetah.JDT
+{
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.Linq;
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// Valid JDT attributes
+    /// </summary>
+    internal enum JdtAttributes
+    {
+        /// <summary>
+        /// Represents an non existant attribute
+        /// </summary>
+        [Description(null)]
+        None = 0,
+
+        /// <summary>
+        /// The JDT path attribute
+        /// </summary>
+        [Description("path")]
+        Path,
+
+        /// <summary>
+        /// The JDT path attribute
+        /// </summary>
+        [Description("value")]
+        Value,
+    }
+
+    /// <summary>
+    /// Validator for JDT attributes within a verb object
+    /// </summary>
+    internal class JdtAttributeValidator
+    {
+        private List<JdtAttributes> validAttributes;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="JdtAttributeValidator"/> class.
+        /// </summary>
+        /// <param name="validAttributes">The attributes that are valid</param>
+        internal JdtAttributeValidator(params JdtAttributes[] validAttributes)
+        {
+            this.validAttributes = validAttributes.ToList();
```

Comment:
Argument validation - it is still possible for `params` style arguments to be null.